content/tutorials/build-apps/build-a-desktop-wallet-in-javascript.md

```diff
@@ -0,0 +1,255 @@
+---
+parent: build-apps.html
+targets:
+  - en
+  - ja # TODO: translate this page
+blurb: Build a graphical desktop wallet for the XRPL using JavaScript.
+---
+# Build a Desktop Wallet in JavaScript
+<!-- STYLE_OVERRIDE: wallet -->
+
+This tutorial demonstrates how to build a desktop wallet for the XRP Ledger using the JavaScript programming language, 
+the Electron Framework and various libraries. This application can be used as a starting point for building a more 
+complex and powerful application, as a reference point for building comparable apps, or as a learning experience to 
+better understand how to integrate XRP Ledger functionality into a larger project.
+
+## Prerequisites
+
+To complete this tutorial, you should meet the following requirements:
+
+- You have node.js 14+ installed on your machine. You can get the latest version [here](https://nodejs.org/en/).
+- You are somewhat familiar with modern JavaScript programming and have completed the [Get Started Using JavaScript tutorial](get-started-using-javascript.html).
+- You have at least some rough understanding of what the XRP Ledger, it's capabilities and of cryptocurrency in general. Ideally you have completed the [Basic XRPL guide](https://learn.xrpl.org/)
+
+### Source Code
+
+You can find the complete source code for all of this tutorial's examples in the [code samples section of this website's repository]({{target.github_forkurl}}/tree/{{target.github_branch}}/content/_code-samples/build-a-wallet/js/).
+
+### Preparing the development machine and installing dependencies
+
+The basic building block for running JavaScript in a non-browser environment, also called headless JavaScript, we need
+to have node.js installed on the development machine.
+
+This tutorial depends on various node.js libraries. To install those dependencies and get the project up and running, in the build-a-wallet-with-javascript/js
+folder, run the following command to install the required modules:
+
+```console
+npm install
+```
```

Comment:
I think it makes more sense to move the installation of dependencies (other than Node itself) under the "Steps" section. And rather than having the user clone the xrpl-dev-portal repo and work from there, I think it would make sense to tell them to make a new project directory somewhere, then show the `package.json` and tell them to copy its contents in before running `npm install`.

Reply:
Done